Gulpfile.js

```diff
@@ -26,20 +26,20 @@ const Root = (aPath = "") => { return gulp.src(RootPath(aPath)); }
 const Src = (aPath = "") => { return gulp.src(SrcPath(aPath)); }
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("clean", done => execTask(_TSC_ + " -b --clean", done));
+gulp.task("clean", done => spawnTask(_TSC_, done, ["--build", "--clean"]));
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("compile", (done) => execTask(_TSC_ + " -b", done));
+gulp.task("compile", (done) => spawnTask(_TSC_, done, ["--build"]));
 
 // ---------------------------------------------------------------------------------------------------------------------
 gulp.task("copy", gulp.parallel(
     () => Root("tsconfig.json").pipe(DistDest()),
-    () => Src(path.join("Config", "**", "*")).pipe(DistDest(path.join("Src", "Config"))),
-    () => Src(path.join("Config", "**", ".*")).pipe(DistDest(path.join("Src", "Config"))),
+    () => Root(path.join("Config", "**", "*")).pipe(DistDest("Config")),
+    () => Root(path.join("Config", "**", ".*")).pipe(DistDest("Config")),
 ));
 
 // ---------------------------------------------------------------------------------------------------------------------
-gulp.task("build", gulp.series("compile", "copy"));
+gulp.task("build", gulp.parallel("compile", "copy"));
```

Comment:
I presume there's no risk `copy` will overwrite `compile` or vice versa? I.e. neither is destroying dirs, just replacing individual files?

Reply:
Correct: if we assume that Config doesn't hold any binary-relevant ts files, we can do these steps in parallel with zero risk.

Reply:
When you say `binary-relevant` ts files, what do you mean? We will have for instance `Config/Globals.ts` that would be imported by `Src/App.ts` is this a problem?

Reply:
I meant "application-relevant", but if we ignore that for moment: I tested this by making a .ts file in `Config` that `App.ts` depends on, and I made sure `Config/` was in the `include` of tsconfig.json. Everything works okay _because_ the compiler doesn't care that the `.ts` has been copied over (potentially ahead of compilation). Remember we're **copying** not **moving**. Hence, we end up with a few extra `.ts` files in `Dist/Config/` but that's no big deal.